### ISSUE TITLE
Getting an error when attempting to open a Lightning channel

### ISSUE THREAD

**Issue by aidenmitchell** [spesmilo/electrum#7169]
`Could not open channel: Exception('LocalConfig. reserve too high: 546, funding_sat: 33421')`
Is there something I'm missing, or is this a bug?

**Comment by SomberNight** [comment]
The bug is that the error message is not user-friendly.

The cause is `funding_sat` being too low. We enforce a minimum of 200_000 sat currently, however your value is so small that it trips an earlier check, which is that the channel reserve cannot be more than 1% of `funding_sat`:
https://github.com/spesmilo/electrum/blob/e686b4081942ee69041338faa277774eb1c1ca7d/electrum/lnutil.py#L103-L104

**Comment by mesca216** [comment]
[]() @msc215

**Comment by aidenmitchell** [comment]
> We enforce a minimum of 200_000 sat currently
Is there a specific reason for this? Can it be changed by the user?

**Comment by SomberNight** [comment]
It is hardcoded here:
https://github.com/spesmilo/electrum/blob/d62a4833e596831b322bdc55d954d35dc4f273ff/electrum/lnutil.py#L312

The reasoning is simply that small capacity channels are problematic for many reasons.

The main issue is that as the onchain fees start to become significant compared to the channel capacity, things start to break down; e.g. the counterparty force-closing the channel costs much of the funds in the channel.
The channel reserve (which is a part of the locked funds that cannot be used for payments) needs to be at least 546 sats
Closing a channel uses, say, 200 vbytes onchain, and if feerates spike to e.g. 100 sat/vbyte then that in itself is already 20_000 sats. The amount that would be used for paying mining fees should the channel get closed at any given time cannot be used for payments either, just like the reserve.
The 200k sat amount that we hardcoded used this napkin math and set the value to one order of magnitude larger than these values. but the limit itself is arbitrary and could potentially be lowered.
